### PR TITLE
bazaar nft market

### PR DESCRIPTION
**Pull request by byyyyyyyyy**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1 +1 @@
-DAPPRADAR_API_KEY=your-sample-key-from-dappradar
+DAPPRADAR_API_KEY=your-sample-key-from-dappradar
```
Comment (liutaurasjanuskas) [inline .env.dist:1]: Please revert this file's changes. Nothing changed here.

File: adapters/bazaar/index.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+class Bazaar {
+    // stands for Ethereum name service
+    constructor() {
+        this.name = "bazaar";
+        this.symbol = "SKILL";
+        this.token = "0x154a9f9cbd3449ad22fdae23044319d6ef2a1fab";
+        this.protocol = "bsc";
+        this.block = 8114624;
+        this.contract = "0x90099dA42806b21128A094C713347C7885aF79e2";
```
Comment (liutaurasjanuskas) [inline adapters/bazaar/index.js:19]: Please lowercase value of `this.contract`

File: .env.dist (hunk 1 above)
Comment (ed3ath) [reply .env.dist:1]: the first commit deleted this file. I just restored it

File: adapters/bazaar/index.js (hunk 2 above)
Comment (ed3ath) [reply adapters/bazaar/index.js]: done

File: adapters/bazaar/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            protocol: this.protocol,
+            nft_contract: this.contract,
+            nft_id: tokenId,
+            token: this.token,
+            token_symbol: this.symbol.symbol,
+            amount: 1,
+            price: nativePrice.toNumber(),
+            price_usd: nativePrice.multipliedBy(po.price).toNumber(),
+            seller: this.contract,
+            buyer,
```
Comment (liutaurasjanuskas) [inline adapters/bazaar/index.js:101]: Please lowercase value of `buyer`.

Comment (ed3ath) [reply adapters/bazaar/index.js]: done

**Pull request merged**